File: Employee.Core/Models/EmployeeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Employee.Core.Models
{
    public class EmployeeModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; } 
        public DateTime DateOfBirth { get; set; }
        public string Gender { get; set; } 
        public string Address1 { get; set; } 
        public string? Address2 { get; set; }
        public string City { get; set; }
        public string Postal { get; set; } 
        public string Country { get; set; }
        public string Email { get; set; } 
        public string Mobile { get; set; } 
        public bool IsActive { get; set; }
    }
}

File: Employee.Angular/src/app/Models/employee.model.ts
export interface Employee {
    id : number;
     firstName: string;
     lastName: string;
     dateOfBirth: Date;
     gender: string;
     address1: string;
     address2?: string;
     city: string;
     postal: string;
     country: string;
     email: string;
     mobile: string;
     isActive: boolean;
};
export interface BaseResponse<T>
{
    responseCode : number,
    responseMessage : string,
    result : T
}

export interface EmployeeTotalModel {
    totalEmployees: number,
    totalActiveEmployees: number,
    totalMaleEmployees: number,
    totalFemaleEmployees: number
}